src/NHSD.BuyingCatalogue.Identity.Api/Settings/ClientSetting.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using IdentityModel;
+using IdentityServer4.Models;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.Settings
+{
+    public sealed class ClientSetting
+    {
+        public string ClientId { get; set; }
+
+        public string ClientName { get; set; }
+
+        public string AllowedGrantTypes { get; set; }
+
+        public int AccessTokenLifetime { get; set; }
```

Comment:
Possibly some indication of unit (seconds), either xml doc comment or rename?
Might also be worth setting the default 3600, as you always override on ToClient.